### DISCUSSION TITLE
Welcome to cf7-to-api Discussions!

### DISCUSSION THREAD

**Discussion by kennym** [kennym/cf7-to-api#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by antoniourraca** [comment]
Hola: 

El plugin ha dejado de funcionar, antes funcionaba ok.

Me da este error en WORDPRESS:

WP_Error Object
(
    [errors] => Array
        (
            [http_request_failed] => Array
                (
                    [0] => cURL error 60: Peer's Certificate issuer is not recognized.
                )

        )

    [error_data] => Array
        (
        )

)
Alguien sabe porque puede ser.

Saludos y gracias de antemano.
